support.html

```diff
@@ -43,6 +43,24 @@ <h2><a href="{{ redHatLink }}">Red Hat</a></h2>
                 </ul>
             </p>
         </section>
+
+        <section class="col-12_md-12">
+            <h2><a href="{{ clysoLink }}">Clyso GmbH</a></h2>
+            <p>
+                As a premium partner of Ceph, Clyso provides expert support, integration, development, and consulting services specifically for Rook and Ceph in large enterprise environments. Clyso’s managed services encompass
+                hybrid cloud solutions utilizing open-source technologies, including Kubernetes, Rook Ceph, and more, ensuring flexibility and eliminating vendor lock-ins.
+                <br />
+                Additionally, Clyso offers advanced backup solutions, comprehensive cloud migration support, and digitalization services through microservices.
+                <br />
+                Services include:
+                <ul>
+                    <li>Managed Ceph and Rook service, consulting and support.</li>
```

Comment:
For consistency, no need for period in the bullets
```suggestion
                    <li>Managed Ceph and Rook service, consulting and support</li>
```